app/views/pages/billing.html

```diff
@@ -463,6 +461,7 @@ <h5 class="title-text">Usage Triggers</h5>
                 <md-checkbox ng-model="settings.db.invoices_by_email" aria-label="Receive Invoices By Email">
                     Receive Invoices By Email
                 </md-checkbox>
+                <a class="cancel-subscribe" ng-click="cancelSubscription($event)" href>Cancel Subscription</a>
                 <div ng-show="$shared.planName()==='pay-as-you-go'">
```

Comment:
Why are we checking for the plan name ?

Reply:
it was previously added. I have not updated.